src/main/java/org/craftercms/studio/impl/v2/service/workflow/WorkflowServiceImpl.java

```diff
@@ -242,8 +243,11 @@ private void createWorkflowEntries(String siteId, List<String> paths, String sub
             throws UserNotFoundException, ServiceLayerException {
         User userObj = userServiceInternal.getUserByIdOrUsername(-1, submittedBy);
         List<Workflow> workflowEntries = new LinkedList<>();
-        paths.forEach(path -> {
+        for (String path : paths) {
             Item it = itemServiceInternal.getItem(siteId, path);
+            if (it == null) {
+                throw new ContentNotFoundException(path, siteId, format("Failed to retrieve item at path '%s' in site '%s'", path, siteId));
```

Comment:
Will that fail the entire publishing queue?

Reply:
It will fail the current request. Notice this is request publish, so they are not in the queue just yet